app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +37,159 @@ class MainActivity : ComponentActivity() {
+                    .background(Color.Black)
+            )
+
+            Box(
+                modifier = modifier
+                    .width(width = 60.dp)
+                    .height(height = 60.dp)
+                    .clip(CircleShape)
+                    .background(Color.Black)
+            )
+        }
+
+        SampleRow(modifier = Modifier.padding(top = 20.dp)) {
+            Box(
+                modifier = Modifier
+                    .size(size = 60.dp)
+                    .border(
+                        width = 2.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(size = 20.dp)
+                    )
+            )
+
+            Box(
+                modifier = Modifier
+                    .size(size = 60.dp)
+                    .border(
+                        width = 3.dp,
+                        color = Color.Black,
+                        shape = CircleShape
+                    )
+            )
+        }
+
+        SampleRow (
+            modifier = Modifier.padding(top = 20.dp)
+        ) {
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .background(Color.Red)
+            )
+
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .clip(CircleShape)
+                    .background(Color.Blue)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 20.dp)
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier.size(size = 80.dp)
+            )
+
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier
+                    .size(size = 80.dp)
+                    .clip(CircleShape)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 10.dp)
+        ) {
+            Column (
+                modifier = modifier
+                    .size(240.dp,140.dp)
+                    .border(
+                        1.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(size = 10.dp)
+                    )
+            ){
+                Column(
+                    modifier = Modifier.padding(16.dp),
+                            verticalArrangement = Arrangement.spacedBy(3.dp)
```

Comment:
검은 선과 쿠잇 이미지 사이의 간격이 6.dp만큼이라 이런 식으로 수정해주세요 ~
```suggestion
                            verticalArrangement = Arrangement.spacedBy(6.dp)
```
그리고 Column안의 아이템이 두 개뿐이라 이런 경우엔 SpacedBy 말고 아이템의padding으로 간격 주는 방식으로 작성하셔도 됩니다 !